tests/pytest/tools/taosdumpTest2.py

```diff
@@ -91,6 +100,30 @@ def run(self):
         tdSql.query("select count(*) from t1")
         tdSql.checkData(0, 0, self.numberOfRecords)
 
+        tdSql.execute("create database test")
+        tdSql.execute("use test")
+        tdSql.execute("create table stb(ts timestamp, c1 binary(16000), c2 binary(16000), c3 binary(10000)) tags(t1 nchar(256))")
```

Comment:
how about using a max boundary value to test the max length row?

Reply:
updated